### PR TITLE
restructuring event consumers to reflect new streams design

### PR DESCRIPTION
**Pull request by andycmaj**
initial re-structuring for adding in `raw_event` handlers

see https://www.notion.so/asynchronous/Streams-and-Events-Implementation-5c83c51913c74e73b41f0b88bbd26302 for design idea

### CODE REVIEW

File: consumers/src/data/apolloClientConfig.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import fetch from 'node-fetch';
 
+// TODO: move to shared
 export default {
   fetch,
-  uri: 'https://botany-db.herokuapp.com/v1alpha1/graphql',
+  uri: process.env.GRAPHQL_URL,
```
Comment (davemeyer) [inline consumers/src/data/apolloClientConfig.ts]: how is this going to get set?

File: shared/cloudEvent.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default interface CloudEvent {
```
Comment (davemeyer) [inline shared/cloudEvent.ts]: are you going to remove the cloudevent sdk package?

File: consumers/src/data/apolloClientConfig.ts (hunk 1 above)
Comment (andycmaj) [reply consumers/src/data/apolloClientConfig.ts]: .env for local already has it... i'll make sure i add to the helm templates.

File: shared/cloudEvent.ts (hunk 2 above)
Comment (andycmaj) [reply shared/cloudEvent.ts]: well that package doesn't have typescript types, just parsers/validators/etc... i guess we could remove it...

Comment (andycmaj) [reply shared/cloudEvent.ts]: actually we do use it to build cloud events using `CloudEvent.event()` so i'll leave it for now

**Pull request merged**